README.md

```diff
@@ -218,9 +218,14 @@ Find the ACORD year/month at the bottom left corner of each ACORD page. Often th
 
 - [CMS 1500 (08/05)](./health_insurance_claims/cms_1500/cms_1500_08_05.json)
 
-### HR
+### hr
```

Comment:
We can leave it capitalized in the readme! I was more talking about the file structure, since the convention there is all lowercase naming for the folders. But here we can use normal English practices.